cmd/argocd/commands/app.go

```diff
@@ -593,7 +596,8 @@ type appOptions struct {
 
 func addAppFlags(command *cobra.Command, opts *appOptions) {
 	command.Flags().StringVar(&opts.repoURL, "repo", "", "Repository URL, ignored if a file is set")
-	command.Flags().StringVar(&opts.appPath, "path", "", "Path in repository to the ksonnet app directory, ignored if a file is set")
+	command.Flags().StringVar(&opts.appPath, "path", "", "Path in repository to the app directory, ignored if a file is set")
+	command.Flags().StringVar(&opts.chart, "chart", "", "Chart is a Helm Chart name")
```

Comment:
maybe `--helm-chart`?

Reply:
agree. This is more in line with other helm related flags. Changed